fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -102,7 +103,17 @@ materializedViewStatement
     | CANCEL MATERIALIZED VIEW TASK taskId=INTEGER_VALUE ON mvName=multipartIdentifier          #cancelMTMVTask
     | SHOW CREATE MATERIALIZED VIEW mvName=multipartIdentifier                                  #showCreateMTMV
     ;
-
+jobScheduleStatement
+    : CREATE JOB label=multipartIdentifier ON SCHEDULE
```

Comment:
remove create job syntax in `sql_parser.cup` and remove `CreateJobStmt.java`

Reply:
I want to directly pick to 2.1, as 2.1 allows for a rollback. Therefore, this change will be provided in the next PR.:)